community/contributing/bisect_workflow.rst

```diff
@@ -0,0 +1,136 @@
+.. _doc_bisect_workflow:
+
+Bisect workflow
+===============
+
+.. highlight:: shell
+
+Bisecting is a way to find regressions in software. After reporting a bug on the
+`Godot repository on GitHub <https://github.com/godotengine/godot>`__, you may
+be asked by a contributor to *bisect* the issue. Bisecting makes it possible for
+contributors to fix bugs faster, as they can know in advance which commit caused
+the regression. Your effort will be widely appreciated :)
+
+The guide below explains how to find a regression by bisecting.
+
+What is bisecting?
+------------------
+
+Godot developers use the `Git <https://git-scm.com/>`__ version control system.
+In the context of Git, bisecting is the process of performing a manual
+`binary search <https://en.wikipedia.org/wiki/Binary_search_algorithm>`__
+to determine when a regression appeared. While it's typically used for bugs,
+it can also be used to find other kinds of unexpected changes such as
+performance regressions.
+
+Using official builds to speed up bisecting
+-------------------------------------------
+
+Before using Git's ``bisect`` command, we strongly recommend trying to reproduce
+the bug with an older (or newer) official release. This greatly reduces the
+range of commits that potentially need to be built from source and tested.
+You can find binaries of official releases, as well as alphas, betas,
+and release candidates `here <https://downloads.tuxfamily.org/godotengine/>`__.
+
+For example, if you've reported a bug against Godot 3.2 RC4, you should first
+try to reproduce the bug in Godot 3.1 (not a patch release, see below for the
+reason). If the bug doesn't occur there, try to reproduce it in Godot 3.2 alpha
+1. If the bug doesn't occur there either, the regression appeared between Godot
+3.1 and 3.2 alpha 1. Otherwise, it appeared in a newer pre-release build; you
+should now test newer alphas, betas and RC builds.
```

Comment:
I'd encourage a more bisect-y approach here, testing intermediate builds first instead of start with alpha 1 and going all the way up to RC 4.
Typically after testing 3.1, I'd test 3.2 beta 1, and then see if I go up or down depending on the result.